package.json

```diff
@@ -70,6 +70,7 @@
     "@typescript-eslint/eslint-plugin": "^5.62.0",
     "@typescript-eslint/parser": "^5.62.0",
     "axe-core": "^4.7.2",
+    "bufferutil": "^4.0.8",
```

Comment:
i'm guessing these packages aren't actually needed?

Reply:
When I installed them - this plus the utf-8-validate one, it removed the console errors that kept showing - so maybe we do?

Reply:
Unless they're specified as peer dependencies for a package, we shouldn't need them.

Which package was requiring `bufferutil` and `utf-8-validate`?

Reply:
i uninstalled the package and there error wasn't recreated so who knows what was going on with that